ConfigConvars.cs

```diff
@@ -37,6 +37,15 @@ public void MatchZyPlayoutConvar(CCSPlayerController? player, CommandInfo comman
             isPlayOutEnabled = bool.TryParse(args, out bool isPlayOutEnabledValue) ? isPlayOutEnabledValue : args != "0" && isPlayOutEnabled;
         }
 
+        [ConsoleCommand("matchzy_save_nades_as_global_enabled", "Whether nades should be saved globally instead of being privated to players by default or not. Default value: false")]
+        public void MatchZySaveNadesAsGlobalConvar(CCSPlayerController? player, CommandInfo command)
+        {
+            if (player != null) return;
+            string args = command.ArgString;
+
+            isPlayOutEnabled = bool.TryParse(args, out bool isPlayOutEnabledValue) ? isPlayOutEnabledValue : args != "0" && isPlayOutEnabled;
```

Comment:
`isPlayOutEnabled` is for a different ConVar. I think you wanted to use `isSaveNadesAsGlobalEnabled` here

Reply:
oh yeah, lack of sleep really shows lately. good catch!